# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/pycups/hotfix-setup.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- pycups-2.0.4/setup.py.vanilla	2026-10-06 18:30:06.907517123 +0300
+++ pycups-2.0.4/setup.py	2026-10-06 18:30:19.003954705 +0300
@@ -34,7 +34,7 @@
 EPSON-Stylus-D78 usb://EPSON/Stylus%20D78
 """
 
-from distutils.core import setup, Extension
+from setuptools import setup, Extension
 import sys
 VERSION="2.0.4"
 libraries=["cups"]
